<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgTimeInterval</class>
 <widget class="QDialog" name="DlgTimeInterval">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>时段表</string>
  </property>
  <widget class="QSpinBox" name="spintimingnum">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>312</x>
     <y>237</y>
     <width>441</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>*要执行的配时方案号。0表示没有可执行方式，即由信号机自主决定控制方式</string>
   </property>
  </widget>
  <widget class="QLabel" name="hang">
   <property name="geometry">
    <rect>
     <x>335</x>
     <y>290</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSpinBox" name="spintimetab">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>16</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinstarthour">
   <property name="geometry">
    <rect>
     <x>289</x>
     <y>79</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="maximum">
    <number>23</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>80</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>开始执行时刻的整分数</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>671</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;黑体&quot;;</string>
   </property>
   <property name="title">
    <string>控制方式</string>
   </property>
   <widget class="QRadioButton" name="radionum8">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>[8]单点优化</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radionum13">
    <property name="geometry">
     <rect>
      <x>454</x>
      <y>60</y>
      <width>114</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>[13]干预线控</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radionum11">
    <property name="geometry">
     <rect>
      <x>216</x>
      <y>60</y>
      <width>114</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>[11]主从线控</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radionum12">
    <property name="geometry">
     <rect>
      <x>335</x>
      <y>60</y>
      <width>114</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>[12]系统优化</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radionum0">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>27</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>[0]自主控制</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radionum3">
    <property name="geometry">
     <rect>
      <x>364</x>
      <y>27</y>
      <width>79</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>[3]全红</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radionum1">
    <property name="geometry">
     <rect>
      <x>196</x>
      <y>27</y>
      <width>79</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>[1]关灯</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radionum6">
    <property name="geometry">
     <rect>
      <x>448</x>
      <y>27</y>
      <width>79</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>[6]感应</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radionum2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>27</y>
      <width>79</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>[2]闪光</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>532</x>
      <y>27</y>
      <width>89</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>置空</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>20</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>时段事件号</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>61</y>
     <width>681</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>时段表号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearall">
   <property name="geometry">
    <rect>
     <x>544</x>
     <y>284</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>清空
全表</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>配时方案号</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spineventnum">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>20</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>48</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>290</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>已设时段表数</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>79</x>
     <y>79</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>开始执行时刻的整点数</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinstartminite">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>79</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="maximum">
    <number>59</number>
   </property>
  </widget>
  <widget class="QPushButton" name="query">
   <property name="geometry">
    <rect>
     <x>463</x>
     <y>284</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>查询
已设</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>271</y>
     <width>681</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="display">
   <property name="geometry">
    <rect>
     <x>342</x>
     <y>336</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="butfind">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>377</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>查 询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butok">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>377</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>修 改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butclear">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>377</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>清空行</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
